src/spec-configuration/containerFeaturesOCI.ts

```diff
@@ -13,103 +13,102 @@ export const DEVCONTAINER_COLLECTION_LAYER_MEDIATYPE = 'application/vnd.devconta
 
 // ghcr.io/devcontainers/features/go:1.0.0
 export interface OCIFeatureRef {
-	registry: string; 		// 'ghcr.io'
-	owner: string;			// 'devcontainers'
-	namespace: string;		// 'devcontainers/features'
-	path: string;			// 'devcontainers/features/go'
-	resource: string;		// 'ghcr.io/devcontainers/features/go'
-	id: string;				// 'go'
-	version?: string;		// '1.0.0'
+    registry: string; 		// 'ghcr.io'
+    owner: string;			// 'devcontainers'
+    namespace: string;		// 'devcontainers/features'
+    path: string;			// 'devcontainers/features/go'
+    resource: string;		// 'ghcr.io/devcontainers/features/go'
+    id: string;				// 'go'
+    version?: string;		// '1.0.0'
 }
 
 // ghcr.io/devcontainers/features:latest
 export interface OCIFeatureCollectionRef {
-	registry: string;		// 'ghcr.io'
-	path: string;			// 'devcontainers/features'
-	version: 'latest';		// 'latest'
+    registry: string;		// 'ghcr.io'
+    path: string;			// 'devcontainers/features'
+    version: 'latest';		// 'latest'
 }
 
 export interface OCILayer {
-	mediaType: string;
-	digest: string;
-	size: number;
-	annotations: {
-		'org.opencontainers.image.title': string;
-	};
+    mediaType: string;
+    digest: string;
+    size: number;
+    annotations: {
+        'org.opencontainers.image.title': string;
+    };
 }
 export interface OCIManifest {
-	digest?: string;
-	schemaVersion: number;
-	mediaType: string;
-	config: {
-		digest: string;
-		mediaType: string;
-		size: number;
-	};
-	layers: OCILayer[];
-	annotations?: {};
+    digest?: string;
+    schemaVersion: number;
+    mediaType: string;
+    config: {
+        digest: string;
+        mediaType: string;
+        size: number;
+    };
+    layers: OCILayer[];
+    annotations?: {};
 }
 
 export function getOCIFeatureSet(output: Log, identifier: string, options: boolean | string | Record<string, boolean | string | undefined>, manifest: OCIManifest): FeatureSet {
 
-	const featureRef = getFeatureRef(output, identifier);
-
-	const feat = {
-		id: featureRef.id,
-		name: featureRef.id,
-		included: true,
-		value: options
-	};
-
-	let featureSet: FeatureSet = {
-		sourceInformation: {
-			type: 'oci',
-			manifest: manifest,
-			featureRef: featureRef,
-
-		},
-		features: [feat],
-	};
-
-	return featureSet;
+    const featureRef = getFeatureRef(output, identifier);
+
+    const feat = {
+        id: featureRef.id,
+        name: featureRef.id,
+        included: true,
+        value: options
+    };
+
+    let featureSet: FeatureSet = {
+        sourceInformation: {
+            type: 'oci',
```

Comment:
Can all the whitespace changes be reverted?

Reply:
Note that i'm trying to make all new files I had use tabs instead of spaces for whitespace, to conform to how vscode/the rest of the codebase is set up.

Reply:
Reverted